### DISCUSSION TITLE
Remove private content from play.tailwindcss.com

### DISCUSSION THREAD

**Discussion by wolfzeru99** [tailwindlabs/tailwindcss#10268] [Help] [answered]
I am quite new into the play.tailwindcss.com.
I have shared some private stuff that I shouldn't share via play.tailwindcss.com.
I thought it would be a chance to overwrite the content and It wasn't so obvious that after clicking share button the content would be permanently saved.
Is there any chance to remove  the content with a specific ID after the backslash?

**Comment by adamwathan** [comment, marked as answer]
Yep happy to do that, email support@tailwindui.com with the ID and I can handle that for you 👍🏻
